### PR TITLE
fix(modifier-set): update modifier-set

### PR DESCRIPTION
**Pull request by mostaphaRoudsari**
Updated modifier-set to match the latest changes in `honeybee-core`.

### CODE REVIEW

File: honeybee_radiance/lib/modifiers.py [hunk 1]
```diff
@@ -1,17 +1,39 @@
 """Library of default modifiers for honeybee-radiance.
 
-Default values are generic values to set the initial visible reflectance and
-transmittance values in your model. There is no guarantee that these values exactly matches your model.
+Default values are generic values to set the initial visible / solar reflectance and
+transmittance values in your model. There is no guarantee that these values exactly
+matches your model.
```
Comment (chriswmackey) [inline honeybee_radiance/lib/modifiers.py]: "match what you are trying to model?"
I am pretty sure that the match the model this is built/exported.

File: honeybee_radiance/modifierset.py [hunk 2]
```diff
@@ -29,13 +31,19 @@
...
+        'skylight': generic_exterior_glass
     },
     'DoorSet': {
-        'exterior': generic_exterior_wall,
-        'interior': generic_interior_wall,
+        'exterior': generic_opaque_door,
+        'interior': generic_opaque_door,
+        'exterior_glass': generic_exterior_glass_door,
+        'interior_glass': generic_interior_glass_door,
         'overhead': generic_interior_wall
```
Comment (chriswmackey) [inline honeybee_radiance/modifierset.py]: Why not use generic_opaque_door here?

File: honeybee_radiance/modifierset.py [hunk 3]
```diff
@@ -218,34 +225,56 @@ class ApertureSet(_BaseSet):
...
-    def __init__(self, exterior_modifier=None, interior_modifier=None,
-                 skylight_modifier=None, glass_door_modifier=None):
+    def __init__(self, window_modifier=None, interior_modifier=None,
+                 skylight_modifier=None, operable_modifier=None):
         """Initialize aperture set.
 
         Args:
-            exterior_modifier: A window modifier object for apertures
+            window_modifier: A window modifier object for apertures
                 with an Outdoors boundary condition.
```
Comment (chriswmackey) [inline honeybee_radiance/modifierset.py]: An Outdoors boundary condition and Wall parent Face.

File: honeybee_radiance/modifierset.py [hunk 4]
```diff
@@ -218,34 +225,56 @@ class ApertureSet(_BaseSet):
...
             interior_modifier: A window modifier object for apertures
                 with a Surface boundary condition.
             skylight_modifier: : A window modifier object for apertures with a
                 Outdoors boundary condition, and RoofCeiling or Floor face type for
                 their parent face.
-            glass_door_modifier: A window modifier object for apertures
+            operable_modifier: A window modifier object for apertures
                 with an Outdoors boundary condition and GlassDoor aperture type.
```
Comment (chriswmackey) [inline honeybee_radiance/modifierset.py]: With an Outdoors boundary condition and a True is_operable property.

**Comment by mostaphaRoudsari** [comment]
Hey Chris! Thank you for the comments. I addressed all of them.

**Pull request merged**